tests/integration/high_availability/scripts/destroy_chaos_mesh.sh

```diff
@@ -1,45 +1,53 @@
-	for i in $(kubectl api-resources | grep chaos-mesh | awk '{print $1}'); do timeout 30 kubectl delete ${i} --all --all-namespaces || :; done
+	for i in $(kubectl api-resources | awk '/chaos-mesh/ {print $1}'); do
+	    timeout 30 kubectl delete "${i}" --all --all-namespaces || true
+	done
 
-	if [ "$(kubectl -n ${chaos_mesh_ns} get mutatingwebhookconfiguration | grep 'choas-mesh-mutation' | wc -l)" = "1" ]; then
-		timeout 30 kubectl -n ${chaos_mesh_ns} delete mutatingwebhookconfiguration chaos-mesh-mutation || :
+	if kubectl -n "${chaos_mesh_ns}" get mutatingwebhookconfiguration | grep -q 'choas-mesh-mutation'; then
+		timeout 30 kubectl -n "${chaos_mesh_ns}" delete mutatingwebhookconfiguration chaos-mesh-mutation || true
 	fi
 
-	if [ "$(kubectl -n ${chaos_mesh_ns} get validatingwebhookconfiguration | grep 'chaos-mesh-validation' | wc -l)" = "1" ]; then
-		timeout 30 kubectl -n ${chaos_mesh_ns} delete validatingwebhookconfiguration chaos-mesh-validation || :
+	if kubectl -n "${chaos_mesh_ns}" get validatingwebhookconfiguration | grep -q 'chaos-mesh-validation'; then
+		timeout 30 kubectl -n "${chaos_mesh_ns}" delete validatingwebhookconfiguration chaos-mesh-validation || true
 	fi
 
-	if [ "$(kubectl -n ${chaos_mesh_ns}) get validatingwebhookconfiguration | grep 'chaos-mesh-validate-auth' | wc -l)" = "1" ]; then
-		timeout 30 kubectl -n ${chaos_mesh_ns} delete validatingwebhookconfiguration chaos-mesh-validate-auth || :
+	if kubectl -n "${chaos_mesh_ns}" get validatingwebhookconfiguration | grep -q 'chaos-mesh-validate-auth'; then
+		timeout 30 kubectl -n "${chaos_mesh_ns}" delete validatingwebhookconfiguration chaos-mesh-validate-auth || true
 	fi
 
-	if [ "$(kubectl get clusterrolebinding | grep 'chaos-mesh' | awk '{print $1}' | wc -l)" != "0" ]; then
+	if kubectl get clusterrolebinding | grep -q 'chaos-mesh'; then
 		echo "deleting clusterrolebindings"
-		timeout 30 kubectl delete clusterrolebinding $(kubectl get clusterrolebinding | grep 'chaos-mesh' | awk '{print $1}') || :
+		read -ra args < <(kubectl get clusterrolebinding | awk '/chaos-mesh/ {print $1}')
+		timeout 30 kubectl delete clusterrolebinding "${args[@]}" || true
 	fi
 
-	if [ "$(kubectl get clusterrole | grep 'chaos-mesh' | awk '{print $1}' | wc -l)" != "0" ]; then
+	if kubectl get clusterrole | grep -q 'chaos-mesh'; then
 		echo "deleting clusterroles"
-		timeout 30 kubectl delete clusterrole $(kubectl get clusterrole | grep 'chaos-mesh' | awk '{print $1}') || :
+		read -ra args < <(kubectl get clusterrole | awk '/chaos-mesh/ {print $1}')
+		timeout 30 kubectl delete clusterrole "${args[@]}" || true
 	fi
 
-	if [ "$(kubectl get crd | grep 'chaos-mesh.org' | awk '{print $1}' | wc -l)" != "0" ]; then
+	if kubectl get crd | grep -q 'chaos-mesh.org'; then
 		echo "deleting crds"
-		timeout 30 kubectl delete crd $(kubectl get crd | grep 'chaos-mesh.org' | awk '{print $1}') || :
+		read -ra args < <(kubectl get crd | awk '/chaos-mesh.org/ {print $1}')
```

Comment:
that line evaluate only the first arg, please check
```
bash-5.2$ cat /tmp/ada
123
chaos-mesh.org1
chaos-mesh.org2
456
789
bash-5.2$ cat /tmp/ada | awk '/chaos-mesh.org/ {print $1}'
chaos-mesh.org1
chaos-mesh.org2
bash-5.2$ read -ra args < <(cat /tmp/ada | awk '/chaos-mesh.org/ {print $1}')
bash-5.2$ echo "${args[@]}"
chaos-mesh.org1
bash-5.2$
```

Reply:
Oh... now I see your comment. Good catch. Fixed in https://github.com/canonical/mysql-k8s-operator/pull/131/commits/44289c3b5c5a48b22c119642c5223abcc8d49da1